examples/stable_diffusion_xl/gm/modules/attention.py

```diff
@@ -132,7 +132,7 @@ def construct(self, x, context=None, mask=None, additional_tokens=None):
         head_dim = q.shape[-1]
         if q_n % 16 == 0 and k_n % 16 == 0 and head_dim <= 256:
             if mask is None:
-                mask = ops.zeros((q_b, q_n, q_n), q.dtype)
+                mask = ops.zeros((q_b, q_n, q_n), ms.uint8)
```

Comment:
will it lead to data type unmatch error?

Reply:
> will it lead to data type unmatch error?

No, this is a limitation of the interface